pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/util/ElasticTestUtil.java

```diff
@@ -76,14 +92,37 @@ public static void deleteIndex(String esIndex) {
         }
     }
 
+    /**
+     * Updates the mappings for the index.  When creating an index for the first time,
+     * query it from an existing elastic instance by running a command like this
+     */
```

Comment:
Like what?

Reply:
Whoops, I moved the guts of this to the createIndex method.  Updating!